package.json

```diff
@@ -22,7 +23,7 @@
 	"dependencies": {
 		"@types/mongoose": "^5.10.5",
 		"@types/node": "^15.0.2",
-		"discord.js": "^12.5.3",
+		"discord.js": "github:discordjs/discord.js",
```

Comment:
We need to create a new issue to make sure we update to the proper v13 once it's released but this is fine for now obviously.

Reply:
#50